.travis/osx_openresty_runner.sh

```diff
@@ -23,8 +23,8 @@ export_or_prefix() {
 }
 
 before_install() {
-    HOMEBREW_NO_AUTO_UPDATE=1 brew install perl cpanminus etcd luarocks openresty/brew/openresty-debug redis@3.2
-    brew upgrade go
+    HOMEBREW_NO_AUTO_UPDATE=1 brew install perl cpanminus etcd luarocks openresty/brew/openresty-debug redis@4.0
+    brew install go@1.13
```

Comment:
Is this modification related to this ssl bug?

Reply:
no, travis test failed, just for test